packages/jaeger-ui/src/components/App/index.jsx

```diff
@@ -38,10 +40,43 @@ import prefixUrl from '../../utils/prefix-url';
 
 import '../common/vars.css';
 import '../common/utils.css';
+import 'antd/dist/reset.css';
 import './index.css';
 import { history, store } from '../../utils/configure-store';
 import { HistoryProvider } from '../../utils/useHistory';
 
+const jaegerTheme = {
```

Comment:
This feels like a possibility to support light/dark themes in the future. Since you were working on this now, do you have an understanding how that can be done? If so, could you book a ticket with an overview of the approach?

Reply:
Sure. Creating a ticket 🚀